app/templates/admin/permissions/event-roles.hbs

```diff
@@ -17,18 +17,19 @@
     </tr>
   </thead>
   <tbody>
-    {{#each model.roles as |item|}}
+    {{#each model.roles as |role|}}
       <tr>
-        <td>{{item}}</td>
-        {{#each (range 1 6) as |number|}}
+        <td>{{role}}</td>
+        {{#each (range 1 6) as |index|}}
```

Comment:
@niranjan94 Is it okay, if I make a helper to simply iterate `n` times like `{{#times 4}}` as the syntax {{#each range(1 6)}} is deprecated, and defining `|index|`and not using it results in template lint error as well, I think the helper would be quite handy if ever need to simply loop too, Please let me know.